TestProjects/Playground/Playground.Forms.Uwp/MainPage.xaml.cs

```diff
@@ -20,11 +11,19 @@ namespace Playground.Forms.Uwp
     /// <summary>
     /// An empty page that can be used on its own or navigated to within a Frame.
     /// </summary>
-    public sealed partial class MainPage : Page
+    public sealed partial class MainPage : WindowsPage
```

Comment:
Remove duplicate inheritance - this is defined in XAML, so just remove ": WindowsPage" from this line